diagnostic.md

```diff
@@ -94,4 +97,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'temMember['hobby']' = 'exercise'
+'teamMember['name']' = 'Jason'
```

Comment:
teamMember['name'] refers to the object { given: 'Jason', surname: 'Weeks' }.  This is what is known as a "nested object" since it's an object within another object.